contexts/generic

```diff
@@ -209,6 +216,11 @@
             "derivedFrom": {
```

Comment:
should probably be consistent everywhere. since we used this, we can simply and remove the prov prefix elsewhere as well.

Reply:
we haven't used any of the prov keys yet in the existing schemas. so we can still decide perhaps whether we want to keep the prefix or not?

Reply:
removing prefix for all.